website/docs/r/virtual_machine.html.markdown

```diff
@@ -1518,6 +1518,24 @@ When cloning from a template, there are additional requirements in both the reso
 
 You can use the [`vsphere_virtual_machine`][tf-vsphere-virtual-machine-ds] data source, which provides disk attributes, network interface types, SCSI bus types, and the guest ID of the source template, to return this information. See the section on [cloning and customization](#cloning-and-customization) for more information.
 
+## Trusted Platfrom Module
+
+When building or cloning from template, there is a additional option to deploy virtual machine with vTPM hardware. This will only work on windows based machines.
```

Comment:
```suggestion
When creating a virtual machine or cloning one from a template, you have the option to add a virtual Trusted Platfrom Module device. Refer to the requirements in the VMware vSphere [product documentation](https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-security/GUID-6F811A7A-D58B-47B4-84B4-73391D55C268.html).
```

There is also a requirement for EFI, but per the documentation, Linux and Windows are supported.